docs/user-guide/migrating-to-3.0.0.md

```diff
@@ -0,0 +1,77 @@
+# Migrating to v3.0.0
+
+ESLint v3.0.0 is the third major version release. We have made several breaking changes in this release, however, we believe the changes to be small enough that they should not require significant changes for ESLint users. This guide is intended to walk you through the changes
+
+## Dropping Support for Node.js < 4
+
+With ESLint v3.0.0, we are dropping support for Node.js versions prior to 4. If you are using an older version of Node.js, we recommend upgrading to at least Node.js 4 as soon as possible. If you are unable to upgrade to Node.js 4 or higher, then we recommend continuing to use ESLint v2.x until you are ready to upgrade Node.js.
```

Comment:
Maybe it's worth mentioning that we are dropping support because everything below Node 4 is in maintenance mode now.